compiler/gcc-cross-compile.cmake

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2015 Alexandre Pretyman
+# All rights reserved.
+#
+# ------------------------------------------------------------------------------
+#  GCC Based Cross Compiler CMake toolchain file
+#
+#  Variables:
+#   CROSS_COMPILE_TOOLCHAIN_PATH=/path/to/toolchain/bin
+#   CROSS_COMPILE_TOOLCHAIN_PREFIX=arm-unknown-linux-gnueabihf
+#   CROSS_COMPILE_SYSROOT=/path/to/sysroot [optional]
```

Comment:
Ok, for now I will do a manual check. I also have to target RaspberryPi 1, so when I get around to do its toolchain, I will create a `poly_test_string_not_empty` with a way to add an error message and refactor this file